api/src/jvmMain/kotlin/io/kinference/compiler/serialization/JvmArraySerialization.kt

```diff
@@ -0,0 +1,68 @@
+package io.kinference.compiler.serialization
+
+import java.nio.ByteBuffer
+
+actual fun ByteArray.toShortTiledArray(blockSize: Int, blocksNum: Int): Array<ShortArray> {
```

Comment:
These functions look very similar. Can we reuse one common function or something like this?